<ui version="4.0" >
 <class>moleculeViewerForm</class>
 <widget class="QWidget" name="moleculeViewerForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="display" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>228</width>
        <height>158</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Display</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QGridLayout" name="_2" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Quality:</string>
           </property>
           <property name="buddy" >
            <cstring>qualityCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="qualityCombo" >
           <item>
            <property name="text" >
             <string>Low</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Medium</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>High</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Style:</string>
           </property>
           <property name="buddy" >
            <cstring>styleCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="styleCombo" >
           <item>
            <property name="text" >
             <string>Balls and sticks</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Sticks</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Van der Waals</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Wireframe</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Labels:</string>
           </property>
           <property name="buddy" >
            <cstring>styleCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="labelsCombo" >
           <item>
            <property name="text" >
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Atoms numbers</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Element symbols</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Element names</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>148</width>
           <height>43</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="edit" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>228</width>
        <height>158</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Edit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QGridLayout" name="_3" >
         <item row="0" column="1" >
          <widget class="QComboBox" name="elementCombo" />
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Element:</string>
           </property>
           <property name="buddy" >
            <cstring>qualityCombo</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>88</width>
           <height>43</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item rowspan="3" row="0" column="1" >
    <widget class="KalziumGLWidget" native="1" name="glWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>10</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>240</width>
       <height>240</height>
      </size>
     </property>
     <property name="sizeIncrement" >
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>240</width>
       <height>110</height>
      </size>
     </property>
     <property name="title" >
      <string>Statistics</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="nameLabel" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Formula:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="formulaLabel" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Weight:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="weightLabel" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>237</width>
       <height>194</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KalziumGLWidget</class>
   <extends>QWidget</extends>
   <header>../../compoundviewer/kalziumglpart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
